mods/ra/rules/structures.yaml

```diff
@@ -1224,7 +1221,6 @@ APWR:
 	RevealsShroud:
 		Range: 4c0
 	Bib:
-	DeadBuildingState:
```

Comment:
:cry: It is a bit sad to see the features I added over the months removed one by one, but it seems I have been outvoted at https://github.com/OpenRA/OpenRA/issues/3522 in keeping an incomplete set so I guess that is okay then.